### DISCUSSION TITLE
Personal portfolio

### DISCUSSION THREAD

**Discussion by jbergant** [webstudio-is/webstudio-community#83] [Q&A]
Hi,
In the personal portfolio template, changing the header is impossible. I tried with AI by giving a prompt: "Change the heading to Special Designs.", but nothing changed, and there was no error or explanation.
I tried clicking double-clicking, and I've searched all the properties and settings. How can I change the header text in the personal portfolio template? It was the first thing I tried to do in the web studio. 

Kind regards,
Jana

**Comment by kof** [comment]
Have you tried to double click on canvas and edit it right where the text is?

**Reply by jbergant** [reply]
Hi, yes, of course. All other text on the template can be edited, except the header.

**Reply by kof** [reply]
reproduced it, @markxtang any ideas how we got there?

**Reply by kof** [reply]
<img width="245" alt="image" src="https://github.com/webstudio-is/webstudio-community/assets/52824/4fdd391f-5fb8-46a7-aee2-1f369bc7d732">

I got it. Text for those headings comes from the Heading element, but you can't double click because its element is covered by the absolutely positioned  "Button Wrapper", so the only way to edit it there is by selecting Text inside Heading in the navigation and hitting Enter.

@markxtang I doubt this was necessary and certainly isn't a great UX, do you need help figuring this out or can you fix it?

**Reply by markxtang** [reply]
I changed the heading text to `position: relative` with a `z-index: 2` which makes it selectable on click. Now the social icons aren't selectable, you have to go through the navigator on the left to select/edit them.

I understand that it's not the most elegant solution, but I couldn't think of any other way to maintain the layout without splitting the heading element into 2 separate elements (which I think would be bad for accessibility). 

I could revise this heading layout altogether if this solution isn't good.

**Reply by kof** [reply]
Lets see how to do this without bad affects at all.

**Reply by markxtang** [reply]
Site updated

**Comment by kof** [comment]
@jbergant for the update to be visible, you need to clone the template again
